content/ref/python/api_walkthrough.md

```diff
@@ -0,0 +1,264 @@
+---
+title: API Walkthrough
+weight: 1
+---
+
+Learn how to track, share, and manage model artifacts in your machine learning workflows with W&B. This notebook covers using W&B to log experiments, generate reports, and access logged data via the W&B Public API.
```

Comment:
```suggestion
Learn how to track, share, and manage model artifacts in your machine learning workflows with W&B. This notebook covers using W&B to log experiments, generate reports, and access logged data via the W&B Public API.
```
Maybe this should be rewritted to reframe the point of this page as being about the *various* APIs and when to use them?

Reply:
Done!